Sources/screen-capture-permissions/ScreenCapturePermissions.swift

```diff
@@ -2,18 +2,16 @@ import Cocoa
 
 struct ScreenCapturePermissions {
   static func checkPermissions() -> String {
-    guard let stream = CGDisplayStream(
+    let stream = CGDisplayStream(
       dispatchQueueDisplay: CGMainDisplayID(),
       outputWidth: 1,
       outputHeight: 1,
       pixelFormat: Int32(kCVPixelFormatType_32BGRA),
       properties: nil,
       queue: DispatchQueue.global(),
-      handler: { status, displayTime, frameSurface, updateRef in })
-    else {
-      return "false";
-    }
+      handler: nil
```

Comment:
Turns out it needs the handler, otherwise it doesn't prompt for the permission (or checks for that matter)

Reply:
Ugh, that is not documented.

You could at least simplify the handler to `handler: { _, _, _, _ in })`